README.md

```diff
@@ -29,19 +29,21 @@ module "subnets" {
   vpc_default_route_table_id = "${var.vpc_default_route_table_id}"
 }
 ```
 
 ## Variables
 
-|  Name                        |  Default       |  Description                                              | Required |
-|:----------------------------:|:--------------:|:--------------------------------------------------------:|:--------:|
-| namespace                    | ``             | Namespace (e.g. `cp` or `cloudposse`)                    | Yes      |
-| stage                        | ``             | Stage (e.g. `prod`, `dev`, `staging`)                     | Yes      |
-| name                         | ``             | Name  (e.g. `bastion` or `db`)                           | Yes      |
-| region                       | ``             | AWS Region where module should operate (e.g. `us-east-1`)| Yes      |
-| vpc_id                       | ``             | The VPC ID where subnets will be created (e.g. `vpc-aceb2723`)         | Yes      |
-| igw_id                       | ``             | The Internet Gateway ID public route table will point to (e.g. `igw-9c26a123`) | Yes       |
-| vpc_default_route_table_id   | ``             | The scheduling expression. (e.g. cron(0 20 * * ? *) or rate(5 minutes) | No       |
-| availability_zones           | []             | The scheduling expression. (e.g. cron(0 20 * * ? *) or rate(5 minutes) | Yes       |
+|  Name                        |  Default       |  Description                                                                                     | Required |
+|:----------------------------:|:--------------:|:------------------------------------------------------------------------------------------------:|:--------:|
+| namespace                    | ``             | Namespace (e.g. `cp` or `cloudposse`)                                                            | Yes      |
+| stage                        | ``             | Stage (e.g. `prod`, `dev`, `staging`)                                                            | Yes      |
+| name                         | ``             | Name  (e.g. `bastion` or `db`)                                                                   | Yes      |
+| region                       | ``             | AWS Region where module should operate (e.g. `us-east-1`)                                        | Yes      |
+| vpc_id                       | ``             | The VPC ID where subnets will be created (e.g. `vpc-aceb2723`)                                   | Yes      |
+| igw_id                       | ``             | The Internet Gateway ID public route table will point to (e.g. `igw-9c26a123`)                   | Yes      |
+| cidr_block                   | ``             | The base CIDR block which will be divided into subnet CIDR blocks (e.g. `10.0.0.0/16`)           | Yes      |
+| vpc_default_route_table_id   | ``             | The scheduling expression. (e.g. cron(0 20 * * ? *) or rate(5 minutes)                           | No       |
```

Comment:
Please change the description for `vpc_default_route_table_id` to something like this:
```
The default route table for public subnets. Provides access to the Internet. If not set here, will be created.
```